comp/process/apiserver/apiserver_test.go

```diff
@@ -43,13 +50,54 @@ func TestLifecycle(t *testing.T) {
 		fetchonlyimpl.MockModule(),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** For now the creation/retrieval of the auth_token is a bit messy in the Agent codebase. I would suggest using the createAndFetch implementation (from `"github.com/DataDog/datadog-agent/comp/api/authtoken/createandfetchimpl"`) which will, as written, create a token and then read it (it will also save it in a global variable in `pkg/api/util` which will allow to retrieve it with `util.GetAuthToken()`. ‏

This will allow us to avoid calling `util.CreateAndSetAuthToken(pkgconfigsetup.Datadog())` in the `EventuallyWithT` loop.

```suggestion
		createandfetch.Module(),
```

Reply:
Updated the test to use `createandfetch`